server/monitor/src/main/java/org/apache/accumulo/monitor/servlets/TServersServlet.java

```diff
@@ -314,13 +314,22 @@ static void doTserverList(HttpServletRequest req, StringBuilder sb, List<TabletS
     long now = System.currentTimeMillis();
 
     double avgLastContact = 0.;
+    double avgResponseTime = 0.;
+    int count = 0;
     for (TabletServerStatus status : tservers) {
+      count++;
       avgLastContact += (now - status.lastContact);
+      avgResponseTime += status.responseTime;
```

Comment:
What I cannot figure out here is that the avgLastContact is not actually an average...it is never divided by count (which I added).